app/localprojectsmanager.cpp

```diff
@@ -184,12 +185,21 @@ QString LocalProjectsManager::findQgisProjectFile( const QString &projectDir, QS
 {
   QList<QString> foundProjectFiles;
   QDirIterator it( projectDir, QStringList() << QStringLiteral( "*.qgs" ) << QStringLiteral( "*.qgz" ), QDir::Files, QDirIterator::Subdirectories );
+
   while ( it.hasNext() )
   {
     it.next();
     foundProjectFiles << it.filePath();
   }
-  if ( foundProjectFiles.count() == 1 )
+
+  if ( QFile::exists( InputUtils::downloadInProgressFilePath( projectDir ) ) )
+  {
+    // if this is a mergin project and file indicating download in progress is still there
+    // download failed or copying from .temp to project dir failed (app was probably closed meanwhile)
+
+    err = tr( "Download failed, remove and download project again" );
```

Comment:
I suggest "Corrupted project directory"

Reply:
Good idea, I will talk to @mostlyAtNight